halpybot/commands/edsm.py

```diff
@@ -69,11 +69,10 @@ async def cmd_distlookup(ctx: Context, args: List[str], cache_override):
     Aliases: dist
     """
     try:
-        # Parse systems/CMDRs from string
-        list_to_str = " ".join([str(elem) for elem in args])
-        points = list_to_str.split(":", 1)
-        pointa, pointb = "".join(points[0]).strip(), "".join(points[1]).strip()
-    except IndexError:
+        pointa, pointb = [
```

Comment:
[nit] these should be `point_a` and `point_b`

Reply:
Fixed upstream in or around 22f9167